api/python/provisioner/config.py

```diff
@@ -38,6 +38,9 @@
 #  - then rename base.py to config.py
 #  - remove PRVSNR_ prefix
 
+PRVSNR_CORTX_CONFIG_DIR = Path('/opt/seagate/cortx_configs')
+PRVSNR_CORTX_CONFIG_FILE = PRVSNR_CORTX_CONFIG_DIR / 'provisioner_cluster.json'
```

Comment:
I believe here we need to either use something like `PRVSNR_CLUSTER_CONFIG_FILE` or `cortx_cluster.json`. In current it seemed like we introduce some provisioner config to be treated as config for the whole CORTX. Also makes sense to think about some suffixes for both directory and files (e.g. `mini`)

@nitin-seagate @ypise do you have any comments here?

Reply:
PRVSNR_CORTX_CONFIG_DIR --> CORTX_CONFIG_DIR
PRVSNR_CORTX_CONFIG_FILE --> CONFSTORE_CLUSTER_CONFIG

Reply:
@ypise Addressed now.